src/rules/converters/semicolon.ts

```diff
@@ -0,0 +1,52 @@
+import { RuleConverter } from "../converter";
+
+// for reference, see here
+// https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/semi.md
```

Comment:
As with #221: Would you mind removing these comments? Folks can generally find all the linked info on https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/ROADMAP.md. The other converters don't have them, and I don't want to start a precedent of adding extra stuff necessary for them - especially since the rule names sometimes change.

Reply:
Ack.